index.html

```diff
@@ -105,6 +107,29 @@ <h3>
           reaches its maximum capacity and gets flushed.
         </p>
       </dd>
+      <dt>
+        <dfn><code>jitterBufferReceivedPackets</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which will be available under origin trial in
+          Chrome. Counts the total number of packets received by the jitter
+          buffer.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>relativePacketArrivalDelay</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which will be available under origin trial in
+          Chrome. Counts the difference between the expected packet arrival
+          time, based on the packets we have seen so far, and the actual arrival
+          time in milliseconds for each packet received by the jitter buffer.
```

Comment:
For time, we use double and seconds, not unsigned long long and milliseconds.

What is "arrival time"? The time it takes from entering to exiting the jitter buffer, or something else?
How is "expected packet arrival time" calculated? Imagine explaining it to someone who wants to implement this metric.

What does "for each packet received" mean? Are we talking mean value or aggregate value? Does it go up with every packet exiting the jitter buffer?

Reply:
Ack, will change to double and seconds. The metric is cumulative, it goes up with every packet inserted in the jitter buffer.

Arrival time is when a packet arrive to the jitter buffer from the network.

The calculation is very implementation specific and intentionally left vague. Lets say that we have received a packet x at some point in the history with send timestamp t1. Now you get a packet y with send timestamp t2. The expected time between packet x and y is t2 - t1. The actual arrival time can be calculated using a timer. The problem is how to pick a good reference packet x (since it could also have been delayed).

Will update the description to make it more clear.